rootfs/templates/confd_settings.py

```diff
@@ -1,8 +1,11 @@
 import os
 
 # security keys and auth tokens
-SECRET_KEY = '{{ getv "/deis/controller/secretKey" }}'
-BUILDER_KEY = '{{ getv "/deis/controller/builderKey" }}'
+with open('/var/run/secrets/api/builder/auth/builder-key') as f:
+    BUILDER_KEY = f.read()
+
+with open('/var/run/secrets/api/django/secret-key') as f:
+    SECRET_KEY = f.read()
```

Comment:
If I understand correctly, the two get merged settings files get merged to create a single settings object?  I'm only thinking it seems odd to have code in `confd_settings.py` for managing values that are entirely unaffected by confd.  That said, I don't feel strongly enough about it to block this.